spec/aep-68/README.md

```diff
@@ -0,0 +1,53 @@
+---
+aep: 68
+title: "Console - Billing & Usage"
+author: Anil Murty (@anilmurty)
+status: Draft
+type: Standard
+category: Interface
+created: 2025-05-20
+updated: 2025-05-20
+estimated-completion: 2025-06-30
+roadmap: major
+---
+
+## Motivation
+
+As number of credit card users in Akash Console grows, a common request we here is being able to view usage and billing info
+
+## Background
+
+Akash Console support two payment options - Crypto Wallet based and Credit Card based. For Crypto Wallets it is easy enough for the users to see when they used tokens for deployments with a blockchain scan tool like mintscan. For credit card users there is no such tracking avaialble. We do get the invoice data from Stripe that we should be able to pass back to the user.
```

Comment:
We could add the functionality to view the spent over time for custodial wallets as well, it's the same logic and endpoint ultimately

Reply:
Good point. Thanks for reviewing